governance/polls/Offboard COMP-A - November 15, 2021.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard the COMP-A Vault Type - November 15, 2021
+summary: Signal your support or opposition to offboard the COMP-A vault type.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboarding-matic-comp-aave-and-bal/11184
+vote_type: Plurality Voting
+categories:
+   - Collateral
```

Comment:
Apparently there's an "Offboard" category - not essential but I guess we could include it?

https://github.com/makerdao/community/blob/master/governance/polls/meta/categories.json